Gemfile

```diff
@@ -1,27 +1,19 @@
 source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
+# Declare your gem's dependencies in devise_2fa.gemspec.
+# Bundler will treat runtime dependencies like base dependencies, and
+# development dependencies will be added by default to the :development group.
 gemspec
 
-gem 'rdoc'
+# Declare any dependencies that are still in development here instead of in
+# your gemspec. These might include edge Rails or gems from your path or
+# Git. Remember to move these dependencies to your gemspec before releasing
+# your gem to rubygems.org.
 
-group :test do
-  platforms :jruby do
-    gem 'activerecord-jdbcsqlite3-adapter', '>= 1.3.0.beta1'
-  end
+# To use a debugger
+# gem 'byebug', group: [:development, :test]
 
-  platforms :ruby do
-    gem 'sqlite3', '~> 1.3.4'
-  end
-
-  gem 'devise', '~> 4.0'
-  gem 'activerecord', '~> 4.2.6'
-  gem 'mongo'
-
-  gem 'rspec'
-
-  gem 'capybara'
-  gem 'shoulda'
-  gem 'selenium-webdriver'
-
-  gem 'minitest-reporters', '>= 0.5.0'
+group :development, :test do
+  gem 'chromedriver-helper'
```

Comment:
https://github.com/titusfortner/webdrivers instead is the correct one these days